src/utils/API.js

```diff
@@ -0,0 +1,8 @@
+import axios from 'axios';
+
+export default {
+  getGames: async () => {
```

Comment:
I think axios it's ok to be added, but I'm not sure about this call to the RAWG API. Will be only for testing or placeholding some info on the views temporally?

Reply:
Both testing and to have a placeholder in the home view, just so we don't have 10+ cards with static info on the grid.

Reply:
Ok, I was asking because in the end I don't think the frontend will be requesting to RAWG directly (or maybe yes, but it will depend in the future). Only to make it clear.